qutebrowser/mainwindow/tabwidget.py

```diff
@@ -439,11 +445,24 @@ def tabSizeHint(self, index):
             # get scroll buttons as soon as needed.
             size = minimum_size
         else:
+            #TODO: relative size and/or configured one
+            try:
+                tab = objreg.get('tab', scope='tab', window=self._win_id, tab=index)
```

Comment:
I don't think this is going to work - the object registry has different indices (just a global counter for all tabs which always increments) than the index you get here.